<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vMFSC | Bach Nguyen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWindow.qrc">
    <normaloff>:/icons/vMFSC.svg</normaloff>:/icons/vMFSC.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QGroupBox" name="requestGroupBox">
      <property name="title">
       <string>Các yêu cầu chỉ định</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="scanDirLabel">
         <property name="text">
          <string>Thư mục quét:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="scanDirLineEdit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="scanDirBrowserButton">
           <property name="text">
            <string>Chọn...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="regExpLabel">
         <property name="text">
          <string>Biểu thức lọc:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="regExpComboBox">
           <item>
            <property name="text">
             <string>25K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>50K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>100K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>250K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>500K</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1M</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>User</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="regExpLineEdit">
           <property name="text">
            <string>\d{4}(\s+|_+|-+)*(i{0,3}I{0,3}|1|2|3|4|iv|IV)(\s+|_+|-+)*(DB|\x0110B|TB|DN|\x0110N|TN)(\s+|_+|-+)*(TT|tt)?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Phạm vi áp dụng</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="fileFormatLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>pdf</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="fileCheckBox">
         <property name="text">
          <string>Cho tên tệp tin có định dạng:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="folderCheckBox">
         <property name="text">
          <string>Cho tên các thư mục</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="scanButton">
          <property name="text">
           <string>Quét</string>
          </property>
          <property name="icon">
           <iconset resource="MainWindow.qrc">
            <normaloff>:/icons/scan.svg</normaloff>:/icons/scan.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="fileProgressLineEdit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="resultGroupBox">
        <property name="title">
         <string>Kết quả quét</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTableView" name="resultTableView"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="resultTotalLabel">
           <property name="text">
            <string>Số lượng: 0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>[&lt;] Bỏ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Chọn [&gt;]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addFromFileButton">
          <property name="text">
           <string>Chọn từ tệp...[&gt;]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeAllButton">
          <property name="text">
           <string>[&lt;&lt;] Bỏ hết</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addAllButton">
          <property name="text">
           <string>Chọn hết [&gt;&gt;]</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="copyGroupBox">
        <property name="title">
         <string>Lựa chọn để sao chép</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0">
          <widget class="QLabel" name="copyDirLabel">
           <property name="text">
            <string>Sao chép tới:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QTableView" name="copyTableView"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="copyDirLineEdit"/>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="copyDirBrowserButton">
           <property name="text">
            <string>Chọn...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QLabel" name="copyTotalLabel">
           <property name="text">
            <string>Số lượng: 0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="noticeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyButton">
        <property name="text">
         <string>Tiến hành sao chép</string>
        </property>
        <property name="icon">
         <iconset resource="MainWindow.qrc">
          <normaloff>:/icons/start.svg</normaloff>:/icons/start.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
